### DISCUSSION TITLE
Scrollbar auto-hide

### DISCUSSION THREAD

**Discussion by NjlsShade** [linuxdeepin/developer-center#3229] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

Some scrollbars in deepin will auto-hide for esthetic reasons when not in use, while this looks great and is not a problem for experienced users, it can be very confusing for some people. Especially since there seems to be a bug where once the scrollbar auto-hids, it will not reemerge when hovering over the area where the bar should be.

The above means you can not always clearly know if a page has more content than what is visible, and that if you don't have a scroll wheel and are not familiar with trackpad gestures scrolling is not possible.

### OBSERVED RESULT | 观察到的结果

Here is an example of the App Store, however, this affects the File Manager, Control Center, and other apps as well (Sorry for using a gif, MP4 and MKV options are still not working properly [3186](https://github.com/linuxdeepin/developer-center/issues/3186)).
![Record_deepin-home-appstore-client_20220604134409](https://user-images.githubusercontent.com/15788214/172019955-11f0ad1e-688c-4f0f-b499-b07c8b661391.gif)

As you can see, there would be no way of knowing this page has more content than what is visible outside of manually trying to scroll with a gesture or wheel.

### EXPECTED RESULT | 期望的结果

Either make the scrollbars reappear when hovered over in these cases, or remove the auto-hiding as a default functionality (maybe make it a personalization setting in Control Center)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Using the latest deepin 20.6.
App Store 7.4.1.0003
File Manager 5.6.3

**Comment by raspin0** [comment]
Windows has an option to turn off auto-hide. I can imagine it can be annoying for many.

**Comment by BLumia** [comment]
Thanks for the suggestion! I'll mark it as a feature request and see if we can improve on this.

**Comment by meiyixiang** [comment]
Thank you for your feedback. This problem exists in many application Windows. I have reported it to the corresponding product manager for handling
